lib/lyrics.js

```diff
@@ -1,41 +1,78 @@
 const fetch = require('node-fetch');
 const urlencode = require('urlencode');
 const fsx = require('fs-extra');
-const printer = require('./cmd_line_printer');
+const printer  = require('./cmd_line_printer');
 const { getLyrics } = require('genius-lyrics-api');
 const clipboardy = require('clipboardy');
 
+const startOfLyricsArea = '</div></div></div></div><div class="hwc"><div class="BNeawe tAd8D AP7Wnd"><div><div class="BNeawe tAd8D AP7Wnd">';
+//const endOfLyricsArea = '</div></div></div></div></div><div><span class="hwx"><div class="BNeawe uEec3 AP7Wnd">';
+const endOfLyricsArea = /<\/div><\/div><\/div><\/div><\/div><div><span class=".*"><div class="BNeawe uEec3 AP7Wnd">/ig;
+
 async function download (song) {
   let i;
   let attemptStr = '';
   try {
-    attemptStr = `https://genius.com/search?q=${urlencode(`${song.tags.artist} ${song.tags.title}`)}`;
-    const options = {
-      // Getted from here: https://genius.com/api-clients
-      apiKey: '4MvSFEk80r1ZQp2QPRi-lXAREzT1u45JtSRWsXgquUY9_zMtWjf1vZOQy6ES-CWl',
-      title: song.tags.title,
-      artist: song.tags.artist,
-      optimizeQuery: true
-    };
-
-    i = await getLyrics(options);
-    if (i.length === 0) {
-      i = null;
+    attemptStr = `https://www.google.com/search?q=${urlencode(`${song.tags.artist} ${song.tags.title}`)}+lyrics`;
+    i = await fetch(attemptStr);
+    i = await i.textConverted();
+    [, i] = i.split(startOfLyricsArea);
+    [i] = i.split(endOfLyricsArea);
+  } catch (m) {
+    try {
```

Comment:
No uses los try-catch como ifs
https://stackoverflow.com/questions/651619/what-is-the-advantage-of-using-try-catch-versus-if-else